actions/docs-verifier/src/ActionRunner/Program.cs

```diff
@@ -36,11 +36,11 @@ static bool IsInWhatsNewDirectory(PullRequestFile file)
     return false;
 }
 
-List<PullRequestFile> files = (await GitHubPullRequest.GetPullRequestFilesAsync(pullRequestNumber)).ToList();
+List<PullRequestFile> files = (await GitHubPullRequest.GetPullRequestFilesAsync(pullRequestNumber)).Where(f => IsYmlOrMarkdownFile(f) && !IsInWhatsNewDirectory(f)).ToList();
```

Comment:
I moved the "Where" part from the loop here because I'm accessing `files.Any` in the loop, so this should be better. This also allowed me to simplify the logic of stripping yml and md extension since we are now sure that all files are already yml or markdown.